src/org/jitsi/impl/neomedia/rtp/remotebitrateestimator/RemoteBitrateEstimatorAbsSendTime.java

```diff
@@ -0,0 +1,627 @@
+package org.jitsi.impl.neomedia.rtp.remotebitrateestimator;
```

Comment:
Can you please add the license header (see any of the other .java files in the project).